app/assets/v2/css/carousel.css

```diff
@@ -0,0 +1,187 @@
+.carousel {
+  background: white;
+  padding: 5px 0;
+  z-index: 0;
+}
+
+.carousel .row {
+  margin-top: 60px;
+  margin-bottom: 40px;
+}
+
+.carousel .control {
+  border-radius: 100%;
+  background-color: #ffce08;
+  width: 40px;
+  height: 40px;
+
+  position: relative;
+  top: 80px;
+}
+
+.carousel .flex-container {
+  display: flex;
+}
+
+.carousel .prev .control {
+  float: right;
+}
+
+.carousel .next .control {
+  float: left;
+}
+
+.carousel .control i {
+  margin: 12px auto;
+  position: relative;
+  top: 1px;
+}
+
+.carousel .prev .control i {
+  right: 1px;
+}
+
+.carousel .next .control i {
+  left: 1px;
+}
+
+.carousel .avatar {
+  margin-right: 30px;
+  margin-left: 70px;
+}
+
+.carousel .avatar i {
+  color: #36e69a;
+}
+
+.carousel .avatar .circles {
+  position: absolute;
+}
+
+.carousel .avatar .closed {
+  position: relative;
+  top: -38px;
+  left: 98px;
+  font-size: 8px;
+}
+
+.carousel .avatar .circle-small {
+	position: relative;
```

Comment:
inconsistent tab / spacing on this file.  can we clean that up real quick?

Reply:
Haha getting that done !!  Thanks for pointing that out ^_^

Reply:
still looks inconsistent to me